resources/views/service/centreusers/edit.blade.php

```diff
@@ -0,0 +1,147 @@
+@extends('service.layouts.app')
+@section('content')
+
+<div id="container">
+    @include('service.includes.sidebar')
+    <div id="main-content">
+
+        <h1>Edit a Children's Centre Worker</h1>
+
+        <p>Use the form below to amend a Children's Centre Worker's details.</p>
+
+        <!-- ADD NEW POST ROUTE HERE eg. method="POST" action="{{ route('admin.vouchers.storebatch') }}" -->
+        <form role="form" class="styled-form">
+            {!! csrf_field() !!}
+            <div class="horizontal-container">
+                <div>
+                    <label for="name" class="required">Name</label>
+                    <input type="text" id="name" value="{{ $worker->name }}" name="name" class="{{ $errors->has('name') ? 'error' : '' }}" required>
+                </div>
+                <div>
+                    <label for="email" class="required">Email Address</label>
+                    <input type="email" id="email" name="email" value="{{ $worker->email }}" class="{{ $errors->has('email') ? 'error' : '' }}" required>
+                </div>
+                <div class="select">
+                    <label for="worker_centre">Home Centre</label>
+                    <select name="worker_centre" id="worker_centre" class="{{ $errors->has('worker_centre') ? 'error' : '' }}" required>
+                        @foreach($centresBySponsor as $sponsor)
+                        <optgroup label="{{ $sponsor->name }}">
+                            @foreach ($sponsor->centres as $centre)
+                                <option value="{{ $centre->id }}"
+                                        @if($centre->selected === "home")
+                                        SELECTED
+                                        @endif
+                                        @if($centre->selected !== false)
+                                            data-workercentre="{{ $centre->selected }}"
+                                        @endif
+                                >
+                                    {{ $centre->name }}
+                                </option>
+                            @endforeach
+                        </optgroup>
+                        @endforeach
+                    </select>
+                </div>
+                <div class="checkboxes">
+                    <div id="alternatives">
+                        <p><strong>Set Neighbours as Alternatives</strong></p>
+                        <div id="centres" class="checkboxes">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <button type="submit" id="createWorker">Add worker</button>
+        </form>
+        <script>
+            function buildCheckboxes() {
```

Comment:
Would it be better to move to using vue templates for the js related pages? I've not tried - not sure how the laravel integration works but the theory I though was to replace blade stuff?

Reply:
The overhaead for the amount of script would be _insane_.

This is not a project where an SPA is required.